src/components/Hosts/Reprovision.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { useMutation } from "@apollo/client";
+import { Popconfirm, Tooltip } from "antd";
```

Comment:
Can you replace these with leafygreen components?

Reply:
Yes, here are the updated screenshots.

With leafygreen `Tooltip`:

<img width="1677" alt="Screen Shot 2021-11-16 at 9 35 19 AM" src="https://user-images.githubusercontent.com/47064971/142005179-71950c03-d3db-4207-b76d-91a199e47a2e.png">

With leafygreen `Popover`:

<img width="1677" alt="Screen Shot 2021-11-16 at 9 35 55 AM" src="https://user-images.githubusercontent.com/47064971/142005203-af96af45-e2bb-41be-aa89-0e48da870726.png">

Replaced for both `Restart Jasper` & `Reprovision` button.